src/plugins/koops_event.conf

```diff
@@ -30,23 +30,23 @@ EVENT=post-create type=Kerneloops remote!=1
 #EVENT=report_Kerneloops type=Kerneloops
         reporter-kerneloops
 
-EVENT=report_Bugzilla type=Kerneloops
+EVENT=report_Bugzilla type=Kerneloops mce!=non-fatal mce!=fatal
         reporter-bugzilla -b \
                 -F /etc/libreport/plugins/bugzilla_format_kernel.conf
 
 # Send micro report
-EVENT=report_uReport type=Kerneloops
+EVENT=report_uReport type=Kerneloops mce!=non-fatal mce!=fatal
```

Comment:
Good, that's the code what I had on mind, but could you please move it here (`src/plugins/koops_event.conf`)?`EVENT=notify` should stay as it is now, because `abrt-applet` listens for this event and shows the desktop notifications - we definitely want to inform users about new MCEs through desktop notifications.